adoc/admin-ses-integration.adoc

```diff
@@ -0,0 +1,594 @@
+Persistent volume claims work by picking any volume that meets the criteria from a pool.
+In this case we specified any volume with a size of 2G or larger.
+When the claim is removed the recycling policy will be followed. 
++
+
+. Create a pod that uses the persistent volume claim. On the {master_node} run the following: 
++
+
+----
+{prompt.user}``kubectl apply -f - <<*EOF*
+apiVersion: v1
+kind: Pod
+metadata:
+  name: POD_NAME
+spec:
+  containers:
+  - name: CONTAINER_NAME
+    image: IMAGE_NAME
+    volumeMounts:
+    - mountPath: /mnt/rbdvol
+      name: rbdvol
+  volumes:
+  - name: rbdvol
+    persistentVolumeClaim:
+      claimName: PV_NAME
+*EOF*`` 
+----
+. Verify that the pod exists and its status. On the {master_node} run: 
++
+
+----
+{prompt.user}``kubectl get po`` 
+----
+. Once pod is running, check the volume by running on the {master_node} : 
++
+
+----
+{prompt.user}``kubectl exec -it CONTAINER_NAME -- df -k`` ...
+/dev/rbd3               999320      1284    929224   0% /mnt/rbdvol
+...
+----
+
+
+In case you need to delete the pod, run the following command on the {master_node}: 
+
+----
+{prompt.user}``kubectl delete pod CONTAINER_NAME`` 
+----
+
+.Deleting A Pod
+[NOTE]
+====
+When you delete the pod, the persistent volume claim is deleted as well.
+The RBD is not deleted. 
+====
+
+== Using RBD for Dynamic Persistent Volumes
+
+
+The following procedure describes how to attach a pod to a RDB dynamic persistent volume. 
+
+.Procedure: Creating a Pod with RBD in Persistent Volume
+. Retrieve the Ceph admin secret. Get the key value using command ``ceph auth get-key client.admin`` or, from the file [path]``/etc/ceph/ceph.client.admin.keyring`` . 
+. On the {master_node} , apply the configuration that includes the Ceph secret by using `kubectl apply`. Replace `CEPH_SECRET` with your Ceph secret. 
++
+
+----
+{prompt.user}``kubectl apply -f - << *EOF*
+apiVersion: v1
+kind: Secret
+metadata:
+  name: ceph-secret-admin
+type: "kubernetes.io/rbd"
+data:
+  key: "$(echo CEPH_SECRET | base64)"
+*EOF*`` 
+----
+
+. Create Ceph user on the SES cluster. 
++
+
+----
+{prompt.root}``ceph auth get-or-create client.user mon "allow r" osd "allow class-read object_prefix rbd_children, allow rwx pool=RBD_POOL" -o ceph.client.kube.keyring`` 
+----
++
+Replace `RBD_POOL` with the RBD pool name.
+
+. Retrieve the Ceph admin secret. Get the key value using command ``ceph auth get-key client.admin`` or, from the file [path]``/etc/ceph/ceph.client.admin.keyring`` . 
+. On the {master_node} , apply the configuration that includes the Ceph secret by using `kubectl apply`. Replace `CEPH_SECRET` with your Ceph secret. 
++
+
+----
+{prompt.user}``kubectl apply -f - << *EOF*
+apiVersion: v1
+kind: Secret
+metadata:
+  name: ceph-secret-user
+type: "kubernetes.io/rbd"
+data:
+  key: "$(echo CEPH_SECRET | base64)"
+*EOF*`` 
+----
+. Create a volume on the SES cluster: 
++
+
+----
+{prompt.root}``rbd create -s SIZE YOUR_VOLUME`` 
```

Comment:
Do we still need to precreate rbd image? I think for dynamic provisioning, it should be handled by rbd provisioner.

Reply:
No, my mistake, its there because I've copied from another instruction block in same document.

Fixed, thanks.